app/src/main/java/com/karthikb351/vitinfo2/contract/Spotlight.java

```diff
@@ -0,0 +1,64 @@
+package com.karthikb351.vitinfo2.contract;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+import co.uk.rushorm.core.RushObject;
+import co.uk.rushorm.core.annotations.RushList;
+
+/**
+ * Created by kushagra on 23/5/16.
+ */
```

Comment:
Remove this thing and put up the proper Copyright header with your name and email.